conf/docker/dataverse-glassfish/init-container/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:7.2.1511
+MAINTAINER Dataverse (support@dataverse.org)
+
+COPY populate /
+# Install dependencies
+#RUN yum install -y unzip
+RUN yum install -y \
+    # cronie \
+    # java-1.8.0-openjdk-devel \
+    nc \
+    perl \
+    postgresql \
+    sha1sum \
+    unzip \
+    wget
+
+
+VOLUME /usr/local/glassfish4/glassfish/domains/domain1/files
```

Comment:
What's this for?  It doesn't seem like this needs to be in a volume.  Even if it does, it should be done in the kube config.